modal/_utils/function_utils.py

```diff
@@ -190,6 +190,25 @@ def serialized_function(self) -> bytes:
             logger.debug(f"Serializing function for class service function {self.cls.__qualname__} as empty")
             return b""
 
+    def get_cls_vars(self) -> Dict[str, Any]:
+        if self.cls is not None:
+            cls_vars = {
+                attr: getattr(self.cls, attr)
+                for attr in dir(self.cls)
+                if not callable(getattr(self.cls, attr)) and not attr.startswith("__")
+            }
+            return cls_vars
+        return {}
+
+    def get_attrs(self) -> Dict[str, Any]:
```

Comment:
Nit: should this be `get_cls_var_attrs` or something like that?

Reply:
Done